### PR TITLE
Add security group 2023 transparency report.

### PR DESCRIPTION
**Pull request by smithp35**

### CODE REVIEW

**Review by pogo59 — APPROVED** [review]
LGTM

File: llvm/docs/SecurityTransparencyReports.rst [hunk 1]
```diff
@@ -76,3 +76,42 @@ the time of writing this transparency report.
...
+in the Armv8.1-m BTI protection, involving a combination of large switch statements
+and __builtin_unreachable() in the default case.
+
+https://bugs.chromium.org/p/llvm/issues/detail?id=43 reports a dependency
+on an old version of xml2js with a CVE filed against it.
+
+https://bugs.chromium.org/p/llvm/issues/detail?id=45 reports a number of
+dependencies that have had vulnerabilities reported against them.
+
+https://bugs.chromium.org/p/llvm/issues/detail?id=46 is related to issue 43
```
Comment (gburgessiv) [inline llvm/docs/SecurityTransparencyReports.rst:103]: nit: Missing trailing `.`

**Review by gburgessiv — APPROVED** [review]
LGTM % nit - thanks again for working on this!

**Comment by smithp35** [comment]
Apologies managed to mess up my branch when trying to fix the typo. I'll try and fix.

**Comment by smithp35** [comment]
Apologies I'll send a new pull request with the typo fixed. Probably the fastest thing to do than trying to get my local branch back into the same state to recreate the additional commit.

**Pull request closed without merging**